packages/embark/src/lib/core/fs.js

```diff
@@ -144,6 +144,7 @@ function removeSync() {
 }
 
 function anchoredPath(anchor, ...args) {
+  args = args.map(path => path.replace(dappPath(), ""));
```

Comment:
What does this do? Is this a bug we had?

Reply:
It was causing issues when the environment variable set for `DAPP_PATH` was absolute instead of relative.